odoo-shippable/scripts/build-image.sh

```diff
@@ -0,0 +1,227 @@
+
+# Extra software download URLs
+HUB_ARCHIVE="https://github.com/github/hub/releases/download/v2.2.3/hub-linux-${ARCH}-2.2.3.tgz"
+NGROK_ARCHIVE="https://dl.ngrok.com/ngrok_2.0.19_linux_${ARCH}.zip"
+
+# Extra software clone URLs
+ZSH_THEME_REPO="https://gist.github.com/9931af23bbb59e772eec.git"
+OH_MY_ZSH_REPO="https://github.com/robbyrussell/oh-my-zsh.git"
+SPF13_REPO="https://github.com/spf13/spf13-vim.git"
+VIM_OPENERP_REPO="https://github.com/vauxoo/vim-openerp.git"
+HUB_REPO="https://github.com/github/hub.git"
+ODOO_REPO="https://github.com/vauxoo/odoo.git"
+MQT_REPO="https://github.com/vauxoo/maintainer-quality-tools.git"
+GIST_VAUXOO_REPO="https://github.com/vauxoo/gist-vauxoo.git"
+PYLINT_REPO="https://github.com/vauxoo/pylint-conf.git"
+
+DPKG_DEPENDS="postgresql-9.3 postgresql-contrib-9.3 \
+              postgresql-9.5 postgresql-contrib-9.5 \
+              perl-modules make pgbadger pgtune \
+              xsltproc xmlstarlet openssl \
+              poppler-utils antiword p7zip-full \
+              expect-dev mosh bpython bsdtar rsync \
+              ghostscript graphviz openssh-server zsh \
+              lua50 liblua50-dev liblualib50-dev \
+              exuberant-ctags git rake"
+PIP_OPTS="--upgrade \
+          --no-cache-dir"
+PIP_DEPENDS_EXTRA="SOAPpy pyopenssl suds \
+                   pillow qrcode xmltodict M2Crypto \
+                   recaptcha-client egenix-mx-base \
+                   PyWebDAV mygengo pandas numexpr \
+                   ndg-httpsclient pyasn1 line-profiler \
+                   watchdog isort coveralls"
+PIP_DPKG_BUILD_DEPENDS="build-essential \
+                        gfortran \
+                        cython \
+                        python-dev \
+                        libfreetype6-dev \
+                        zlib1g-dev \
+                        libjpeg-dev \
+                        libblas-dev \
+                        liblapack-dev \
+                        libpq-dev \
+                        libldap2-dev \
+                        libsasl2-dev \
+                        libxml2-dev \
+                        libxslt1-dev \
+                        libgeoip-dev"
+NPM_OPTS="-g"
+NPM_DEPENDS="localtunnel \
+             fs-extra"
+
+# Let's add the git-core ppa for having a more up-to-date git
+add_custom_aptsource "${GITCORE_PPA_REPO}" "${GITCORE_PPA_KEY}"
+
+# Release the apt monster!
+apt-get update
+apt-get upgrade
+apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
+
+# Install node dependencies
+npm install ${NPM_OPTS} ${NPM_DEPENDS}
+
+# Fix reinstalling npm packages
+# See https://github.com/npm/npm/issues/9863 for details
+sed -i 's/graceful-fs/fs-extra/g;s/fs.rename/fs.move/g' $(npm root -g)/npm/lib/utils/rename.js
+
+# Install python dependencies
+pip install ${PIP_OPTS} ${PIP_DEPENDS_EXTRA}
+
+# Clone odoo & tools
+git_clone_copy "${ODOO_REPO}" "8.0" "" "${REPO_REQUIREMENTS}/odoo"
+#git_clone_copy "${GIST_VAUXOO_REPO}" "master" "" "/root/tools/gist-vauxoo"
```

Comment:
I was going to ask what to do in this case, because that repo is private and will make the build fail in the docker hub automated build. Is the gist-vauxoo repo necessary? I was planning to leave it commented until #109 is done. What do you think?